### PR TITLE
Added flag to ensure mouse event signals are not connected twice

### PR DESCRIPTION
**Pull request by RebeccaStankus**
https://highfidelity.atlassian.net/browse/DEV-2349

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

File: libraries/entities-renderer/src/EntityTreeRenderer.cpp [hunk 1]
```diff
@@ -157,6 +157,7 @@ render::ItemID EntityTreeRenderer::renderableIdForEntityId(const EntityItemID& i
 }
 
 int EntityTreeRenderer::_entitiesScriptEngineCount = 0;
+bool _signalsConnected = false;
```
Comment (zfox23) [inline libraries/entities-renderer/src/EntityTreeRenderer.cpp]: please make this a private member variable of the `EntityTreeRenderer` class, and name it something slightly more descriptive like `_mouseSignalHandlersConnected`

Comment (RebeccaStankus) [reply libraries/entities-renderer/src/EntityTreeRenderer.cpp]: I can make it private but it does not only include mouse event signals

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21795/HighFidelity-Beta-PR16313-0924245.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21795/HighFidelity-Quest-Beta-PR16313-0924245.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21795/HighFidelity-Beta-PR16313-0924245.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21801/HighFidelity-Beta-PR16313-458d3bf.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21801/HighFidelity-Quest-Beta-PR16313-458d3bf.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21801/HighFidelity-Beta-PR16313-458d3bf.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21801/HighFidelity-Beta-Interface-PR16313-458d3bf.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21801/HighFidelity-Beta-PR16313-458d3bf.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21801/HighFidelity-Beta-Interface-PR16313-458d3bf.exe

**Pull request merged**